src/sdk/api/api.service.ts

```diff
@@ -154,7 +145,7 @@ export class ApiService extends Service {
 
       try {
         // checks and removes outdated cache entries
-        this.cache.expire();
+        // this.cache.expire();
```

Comment:
Do we still need this line?

Reply:
nope since the cache is managed in memory rather than handling by a separate policy package

Reply:
will remove that comment line